Mytinerary/Models/Itinerary.m

```diff
@@ -55,6 +56,17 @@ - (BOOL)isEventDateValid:(NSDate *)eventStartTime eventEndTime:(NSDate *)eventEn
     return true;
 }
 
+
+- (void)updateItinerary:(Itinerary *)updatedItinerary {
```

Comment:
is the itinerary itself being updated in a view controller directly? i would recommend putting the updating of the object (ex. itinerary.title = title) within this function and pass in the new data as arguments, just for the sake of encapsulation. (i did something similar in events.m)